django_rq/views.py

```diff
@@ -223,6 +223,42 @@ def deferred_jobs(request, queue_index):
     return render(request, 'django_rq/jobs.html', context_data)
 
 
+@staff_member_required
+def scheduled_jobs(request, queue_index):
+    if not RQ_SCHEDULER_INSTALLED:
+        return redirect('rq_jobs', queue_index)
+
+    queue_index = int(queue_index)
+    queue = get_queue_by_index(queue_index)
+
+    scheduler = get_scheduler(queue.name)
+    all_jobs = scheduler.get_jobs()
+
+    items_per_page = 100
+    num_jobs = len(all_jobs)
```

Comment:
Instead of fetching all jobs and paginating them in Python, we should use `scheduler.get_jobs(offset=foo, length=bar)`